packages/nextjs-cache-handler/src/helpers/ioredisAdapter.ts

```
@@ -0,0 +1,137 @@
+            } else if (options.PXAT) {
+              args.push("PXAT", options.PXAT);
+            } else if (options.EX) {
+              args.push("EX", options.EX);
+            } else if (options.PX) {
+              args.push("PX", options.PX);
+            } else if (options.KEEPTTL) {
+              args.push("KEEPTTL");
+            }
+            // Add other options if necessary
+          }
+          // Cast to a generic signature to avoid overload mismatch issues with dynamic args
+          const setFn = target.set as unknown as (
+            key: string,
+            value: string | number,
+            ...args: (string | number)[]
+          ) => Promise<string | null>;
+
+          return setFn(
+            args[0] as string,
+            args[1] as string | number,
+            ...args.slice(2),
+          );
+        };
+      }
+
+      if (prop === "hmGet") {
+        return async (key: string, fields: string | string[]) => {
+          const args = Array.isArray(fields) ? fields : [fields];
+          if (args.length === 0) {
+            return [];
+          }
+          return target.hmget(key, ...args);
+        };
+      }
+
+      // Handle camelCase to lowercase mapping for other methods
+      if (typeof prop === "string") {
+        // Special case for expireAt -> expireat
+        if (prop === "expireAt") {
+          return target.expireat.bind(target);
+        }
+
+        // hSet -> hset
+        if (prop === "hSet") {
+          return target.hset.bind(target);
+        }
+
+        // hExists -> hexists
+        if (prop === "hExists") {
+          return target.hexists.bind(target);
+        }
+
+        // Default fallback to lowercase if exists
+        const lowerProp = prop.toLowerCase();
+        if (
+          lowerProp in target &&
+          typeof (target as any)[lowerProp] === "function"
+        ) {
+          return (target as any)[lowerProp].bind(target);
+        }
```

Not sure what it does, camelCase fallback may mask mistakes and typos right?